persistence/src/main/scala/akka/persistence/kafka/journal/KafkaJournal.scala

```diff
@@ -27,9 +28,19 @@ class KafkaJournal(config: Config) extends AsyncWriteJournal { actor =>
 
   implicit val system       : ActorSystem              = context.system
   implicit val executor     : ExecutionContextExecutor = context.dispatcher
-  implicit val contextShift : ContextShift[IO]         = IO.contextShift(executor)
-  implicit val parallel     : Parallel[IO]             = IO.ioParallel(contextShift)
-  implicit val timer        : Timer[IO]                = IO.timer(executor)
+
+  private val (blocking, blockingShutdown)   = IORuntime.createDefaultBlockingExecutionContext("kafka-journal-blocking")
```

Comment:
I checked internals, I think this is fine to use consistently this one instead of custom `Executors.blocking`

Reply:
Good, thanks! As I've already mentioned, the only differences I've spotted is minSize = 0 instead of 2 and daemonic threads instead of non-daemonic